feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.component.ts

```diff
@@ -26,24 +32,26 @@ export class ConfiguratorAttributeQuantityComponent
   quantity = new FormControl(1);
   protected sub: Subscription;
 
-  @Input() allowZero = true;
-  @Input() initialQuantity: Quantity;
-  @Input() disableQuantityActions = false;
+  @Input() quantityOptions: ConfiguratorAttributeQuantityComponentOptions = {
+    allowZero: true,
```

Comment:
Is this default needed (as overwritten by the caller)?

Reply:
Done!